src/guide/testing.md

```diff
@@ -1,165 +1,165 @@
+#### Comunidad Activa y equipo
 
-Since the majority of unit testing frameworks are open-source, having a community that is active can be critical to some teams that will be maintaining their tests for a long period of time and needs to ensure that a project will be actively maintained. In addition, having an active community has the benefit of providing more support whenever you run into issues.
+Como la mayoría de los marcos de trabajo de pruebas unitarias son de código abierto, tener una comunidad que es activa puede ser crítico para algunos equipos que mantendrán sus pruebas por un largo período de tiempo y necesitar estar seguros de que un proyecto será mantenido activamente. Además, tener una comunidad activa tiene el beneficio de proveer más soporte cuando ocurra un problema.
 
-### Frameworks
+### Marcos de Trabajo
 
-While there are many tools in the ecosystem, here are some common unit testing tools that are being used in the Vue.js ecosystem.
+Aunque existen muchas herramientas en el ecosistema, aquí están algunas herramientas comunes de pruebas unitarias que están siendo utilizadas en el ecosistema de Vue.js.
 
 #### Jest
 
-Jest is a JavaScript test framework that is focused on simplicity. One of its unique features is the ability to take snapshots of tests in order to provide an alternative means of verifying units of your application. 
+Jest es un marco de trabajo de pruebas de JavaScript que se enfoca en la simplicidad. Una de sus características únicas es la habilidad de tomar instantáneas de pruebas para proveer un medio alternativo de verificar las unidades de su aplicación.
 
-**Resources:**
+**Recursos:**
 
-- [Official Jest Website](https://jestjs.io)
-- [Official Vue 2 CLI Plugin - Jest](https://cli.vuejs.org/core-plugins/unit-jest.html)
+- [Sitio Web Oficial de Jest](https://jestjs.io)
+- [Plugin Oficial de Línea de Comandos para Vue 2 - Jest](https://cli.vuejs.org/core-plugins/unit-jest.html)
 
 #### Mocha
 
-Mocha is a JavaScript test framework that is focused on being flexible. Because of this flexibility, it allows you to choose different libraries to fulfill other common features such as spying (e.g., Sinon) and assertions (e.g., Chai). Another unique feature of Mocha is that it can also execute tests in the browser in addition to Node.js.
+Mocha es un marco de trabajo de pruebas de JavaScript enfocado en ser flexible. Por esta flexibilidad, le permite elegir diferentes librerías para cumplir con otras características comunes como espiando (p. ej., Sinon) y aserciones (p. ej., Chai). Otra característica única de Mocha es que también puede ejecutar pruebas en el navegador y en Node.js.
 
-**Resources:**
+**Recursos:**
 
-- [Official Mocha Website](https://mochajs.org)
-- [Official Vue CLI Plugin - Mocha](https://cli.vuejs.org/core-plugins/unit-mocha.html)
+- [Sitio Web Oficial de Mocha](https://mochajs.org)
+- [Plugin Oficial de Línea de Comandos para Vue 2 - Mocha](https://cli.vuejs.org/core-plugins/unit-mocha.html)
```

Comment:
Is the version (Vue 2) required? 😮  @lagp0310

Reply:
No, it's a mistake. Thanks @migsalazar! :+1:

Reply:
```suggestion
- [Plugin Oficial de Línea de Comandos - Mocha](https://cli.vuejs.org/core-plugins/unit-mocha.html)
```